packages/editor-ui/src/views/NodeView.vue

```diff
@@ -2760,6 +2760,10 @@ export default mixins(
 				}
 				this.stopLoading();
 
+				if (this.$store.getters['settings/deploymentType'] !== 'n8n-internal') {
+					console.log(HIRING_BANNER); // eslint-disable-line no-console
```

Comment:
can you move this into `App.vue`? this way it will render on all new pages we are adding. and it will only render once then.